tests/xmap_test.py

```diff
@@ -55,6 +55,18 @@
 from jax.config import config
 config.parse_flags_with_absl()
 
+try:
```

Comment:
Can I suggest using `contextlib.suppress(ImportError)` to achieve this rather than try;except?

(same everywhere)

Reply:
Nice, didn't know about that! Updated (and also removed type disable comments, pretty sure I don't need those)